modules/balana-samples/kmarket-trading-sample-web-service/src/main/java/org/wso2/balana/balanasamples/KmarketTradingSampleWebServiceApplication.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.balana.balanasamples;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class KmarketTradingSampleWebServiceApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(KmarketTradingSampleWebServiceApplication.class, args);
```

Comment:
Please add a new line after the method signature and remove other additional new lines in 12 and 14.